specification/compute/resource-manager/Microsoft.Compute/stable/2021-04-01/compute.json

```diff
@@ -10960,6 +11866,10 @@
           "$ref": "#/definitions/StorageProfile",
           "description": "Specifies the storage settings for the virtual machine disks."
         },
+        "capacityReservation": {
+          "$ref": "#/definitions/CapacityReservationProfile",
```

Comment:
Pls. ensure that capacityReservation property is also added in the restore point source VM properties defined in RestorePointSourceMetadata.